tests/sles4sap/hana_install.pm

```diff
@@ -65,10 +110,15 @@ sub run {
         # in a different backend, assume sdb exists. Always create mountpoints.
         foreach (keys %mountpts) { assert_script_run "mkdir -p $mountpts{$_}->{mountpt}"; }
         if ((check_var('BACKEND', 'qemu') and get_var('HDDSIZEGB_2')) or !check_var('BACKEND', 'qemu')) {
-            my $device = (check_var('HDDMODEL', 'scsi-hd') or !check_var('BACKEND', 'qemu')) ? '/dev/sdb' : '/dev/vdb';
-            script_run "wipefs -f $device; wipefs -f ${device}1";
+            my $device = check_var('HDDMODEL', 'scsi-hd') ? '/dev/sdb' : '/dev/vdb';
+            # We need 2.5 times $RAM + 50G for HANA installation.
+            # On qemu, we assume 2nd disk was properly configured in settings
+            # On BACKENDS different than qemu make sure we get a device with at least that size
+            $device = $self->get_hana_device_from_system(($RAM * 2.5) + 50000) unless check_var('BACKEND', 'qemu');
```

Comment:
This command should work on qemu backend also, no? If yes maybe it's easier to execute it in any case? The check with the `HDDMODEL` could be removed also in that case. WDYT?

Reply:
Let's test it on qemu. On it!

Reply:
It works on qemu. I added a verification run below, and will also add a verification run for PowerKVM. Code has been pushed if you want to review again.